docs/howto/create-a-datasource/configuring-the-jdbc-driver-manually.adoc

```diff
@@ -28,6 +28,7 @@
 +
 In the `thorntail` -> `datasources` -> `jdbc-drivers` tree, define the driver by
 key, which will be used later for connecting it to the datasource.
+Set the `driver-module-name` to that defined in the `module.xml`.
```

Comment:
Grammatically, this isn't great (you'd say _defined in `module.xml`_ or, preferably, _defined in the `module.xml` file_), can you please opt for one of those variants? In the case of `driver-module-name`, the situation is a bit different because when you read the instruction out loud, it goes _Set the driver module name,_ which works. Thanks.